vendor/xclannad/wavfile.cc

```diff
@@ -738,7 +742,7 @@ void MP3FILE::Seek(int count) {
 	} while(pimpl->status != MP3FILE_impl::DONE && pimpl->write_pointer < pimpl->write_data_len);
 	return;
 }
-#elif USE_SMPEG
```

Comment:
Probably.

In retrospect, I suspect that mp3 code has never ran in production. It's certainly not built on Debian (see debian/control) and I don't think it's part of the Mac build. VisualArts is an ogg shop and I know of no games of that they publish that use mp3 files.

(They did use mpeg1 files for their videos. Even in the late '00s. I suspect this was some sort of compatibility thing where they could plug into some OS primitive without having to license patents themselves, but that's a guess and I have no direct knowledge of the situation.)

In vendor/xclannad/file.cc, there were similar constructs for PNG and JPEG files, which (AFAIK) VisualArts never used and weren't used in any patches known to me. However, since basically everything with a cpu has libpng on it, I left it. Double checking this now though, it looks like more dead code...because rlvm proper doesn't recognize "png" or "jpg" as valid. I'll do those myself after I merge your patch here.